implementation/core/src/main/java/io/smallrye/faulttolerance/core/timer/ThreadTimer.java

```diff
@@ -89,23 +104,25 @@ public ThreadTimer(Executor defaultExecutor) {
                                     try {
```

Comment:
Unnecessary creation of lambda for each call: Task can implement Runnable and do this without extra allocations.

Reply:
If we allocate any lambdas anywhere, it's not in the timer. The `schedule` methods already accept a `Runnable`.

Reply:
GitHub shows here not the lines I have commented. I speak about

`executorForTask.execute(() -> {
    LOG.runningTimerTask(task);
    try { `

Reply:
Ouch! Understood. Yes, that makes sense, thanks!

Reply:
Done.